manifest.json

```diff
@@ -6,9 +6,9 @@
 		"en": "Create a beautiful blog or website easily",
 		"fr": "Logiciel de création de blog ou de site Web"
 	},
-	"version": "4.8",
+	"version": "4.9.1~ynh1",
```

Comment:
Here our new versioning rule gives us "4.9.1-1", doesn't it?

Reply:
That's not what I understood.
Is there anything new about that ?

After what josue (?) said, I understood it was better to use a tilde and "ynh" to not interfere with the version number of the app.

Reply:
OK, looks like I totally missed [it](https://github.com/YunoHost/doc/commit/25471ccb257bc5ecfb39675a3b5c57821cf57ec5)... so  please forget it 😊